### PR TITLE
Spring Cleaning

### PR DESCRIPTION
**Pull request by CillianMyles**
- Deprecation message for old documentation (`v2.3` & `v3.0`)
- Remove old copy of API docs for `v3` which are broken and no longer used
- Tidied up page titles including main page:
-> "OnePageCRM - Developers Workspace" -> "Developer Workspace"
- Updated README w/ details of new API docs, and how to make changes
- Remove unused config files

### CODE REVIEW

File: api2_deprecated/index.html [hunk 1]
```diff
@@ -1,19 +1,23 @@
 ---
```
Comment (kevGalway) [inline api2_deprecated/index.html]: Can we just delete this file? URL has changed so users can't find it anyway + if we need it we can look through source control

File: api3_deprecated/index.html [hunk 2]
```diff
@@ -1,16 +1,21 @@
 ---
 layout: api3
```
Comment (kevGalway) [inline api3_deprecated/index.html]: Can we just delete this file/folder? URL has changed so users can't find it anyway + if we need it we can look through source control

**Comment by CillianMyles** [comment]
Thanks for the review Kevin :+1: 

- 'old' folder deleted
- I am happy enough to remove the `apiX_deprecated` folders/pages. I really just marked them as deprecated as the first step towards removing them. Will I remove them both or just `api2_`? I am happy to remove both if you are?
- The `*api_browser`s are a different kettle of fish. I'm reluctant to remove these (especially dev ones), since the interactive Swagger docs don't allow us to make requests to dev servers. This can be a very useful tool when developing a feature for mobile, which uses new/undocumented API. PS - I don't think they are linked anywhere. I might be happy enough to delete them if we had another solution whereby we could use interactive Swagger docs with dev servers. What do you think?

**Comment by kevGalway** [comment]
I think we can remove `apiX_deprecated`. It is in version control anyway so we can get it back if needed. 

There is a way to have interactive Swagger docs with dev servers. The servers array at the start of the doc allows section of different base urls. However, we shouldn't just add dev servers directly here as we don't want to show this to everyone. Maybe we can leave this for now. 

Good work! Great to have this cleaned up

**Comment by CillianMyles** [comment]
Sorry only seeing this now. Will remove `apiX_deprecated` folders and merge. Thanks Kevin! :+1:

**Comment by CillianMyles** [comment]
@kevGalway Can you give my recent changes a final once over before I merge?

**Pull request merged**